autoextract_poet/utils.py

```diff
@@ -0,0 +1,22 @@
+from autoextract.aio import request_raw
+from autocrawl.autoextract_retry import autoextract_retry
+
+from autoextract_poet.exceptions import QueryLevelError
+
+
+async def request(url: str, page_type: str, full_html=True):
+    query = [dict(url=url, pageType=page_type, fullHtml=full_html)]
+
+    async def _request():
+        response_per_url = await request_raw(query)
```

Comment:
Shouldn't we send some APIKEY here too?

Reply:
I think it'd be better to take the API key from Scrapy settings, as it'd be consistent with scrapy-autoextract. To do it, request function could accept api_key argument, and a provider may require Settings.

Reply:
Now, Scrapy Settings is listed as a dependency of this provider. 
We're trying to get the API Key from the settings under the key name `SCRAPINGHUB_AUTOEXTRACT_KEY`.